<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-6LGQ6RF" failures="1" tests="6" name="DemoTest" time="12.008" errors="0" timestamp="2022-11-25T15:46:12 IST">
  <testcase classname="demo.advanced.tests.DemoTests" name="ZTest1" time="0.018"/>
  <testcase classname="demo.advanced.tests.DemoTests" name="BTest3" time="0.003"/>
  <testcase classname="demo.advanced.tests.DemoTests" name="ATest2" time="0.006">
    <skipped/>
  </testcase> <!-- ATest2 -->
  <testcase classname="demo.advanced.tests.DemoTests" name="ATest2" time="0.003">
    <skipped/>
  </testcase> <!-- ATest2 -->
  <testcase classname="demo.advanced.tests.DemoTests" name="ATest2" time="0.001">
    <skipped/>
  </testcase> <!-- ATest2 -->
  <testcase classname="demo.advanced.tests.DemoTests" name="ATest2" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.advanced.tests.DemoTests.ATest2(DemoTests.java:41)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- ATest2 -->
</testsuite> <!-- DemoTest -->
